### PR TITLE
Add Block Footer With Lists and Update Block Two Column

### PR DESCRIPTION
**Pull request by chrismekim**
View at `/capabilities`

Preview:
Screen Shot 2020-04-21 at 2 55 14 PM

### CODE REVIEW

File: src/components/BlockTwoColumn.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ const BlockTwoColumn = props => {
   const sectionThreeText = get(fields, 'sectionThreeText', '');
   const sectionFourTitle = get(fields, 'sectionFourTitle', '');
   const sectionFourText = get(fields, 'sectionFourText', '');
+  const hideSectionTextOnMobile = get(fields, 'hideSectionTextOnMobile', false);
```
Comment (chrismekim) [inline src/components/BlockTwoColumn.js:18]: Added the option to hide the section text (paragraph) on mobile.

**Comment by devinhalladay** [comment]
Looking good! Two changes to request:

image
Can we please make these links closer to the image above? like 1rem padding top. And add maybe 10rem padding below them to add whitespace above the footer

image
Can you please add an additional 4rem of spacing above and below the callout block, so it looks like the image above?

**Comment by chrismekim** [comment]
@devinhalladay made those changes!

File: src/components/BlockFooterWithLists.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+          <div className="flex md:col-4 pb2 md:pb0">
+            <List
+              title={listTwoTitle}
+              listItems={simpleFragmentToListItems(listTwo)}
+            />
+          </div>
+        </div>
+      </div>
+      <div className="flex md:flex-row justify-between pt1 md:pt10">
+        <>
```
Comment (limsohee1002) [inline src/components/BlockFooterWithLists.js:46]: Why are you wrapping this with fragment?

**Pull request merged**